pkgs/applications/networking/sync/unison/default.nix

```diff
@@ -26,9 +25,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
-  nativeBuildInputs = [ glib wrapGAppsHook ocamlPackages.ocaml ]
-    ++ lib.optional enableX11 copyDesktopItems;
-  buildInputs = [ gsettings-desktop-schemas ncurses zlib ]
+  nativeBuildInputs = [ ocamlPackages.ocaml ]
+    ++ lib.optionals enableX11 [ copyDesktopItems wrapGAppsHook ];
+  buildInputs = [ ncurses zlib ]
+    ++ lib.optional enableX11 gsettings-desktop-schemas
     ++ lib.optional stdenv.isDarwin Cocoa;
```

Comment:
Nit: Unless there is a deeper meaning behind there being exactly one element in the list, I prefer `optionals [ ... ]`.